tests/cases/fourslash/completionForStringLiteralRelativeImport1.ts

```diff
@@ -51,20 +52,21 @@ for (const kind of kinds) {
 
     goTo.marker(kind + "1");
     verify.completionListContains("f1");
-    verify.completionListContains("f2");
-    verify.completionListContains("e1");
-    verify.completionListContains("folder");
-    verify.completionListContains("parentTest");
-    verify.not.completionListItemsCountIsGreaterThan(5);
+    verify.completionListContains("f3");
+    verify.completionListContains("f4");
+    verify.completionListContains("f7");
+    verify.completionListContains("d1");
+    verify.completionListContains("d2");
+    verify.not.completionListItemsCountIsGreaterThan(6);
```

Comment:
The interface (viewable in stc/fourslash/fourslash.ts) doesn't contain such a method, but I can add it if you think it's a good idea. If so, should I do that in a separate PR?